services/adgs/config/adgs_ws_config.yaml

```diff
@@ -35,98 +40,96 @@ adgs:
+      - $.ContentDate.End
+      startTimeFromAscendingNode:
+      - null
+      - $.PublicationDate
+      completionTimeFromAscendingNode:
+      - null
+      - $.PublicationDate
+      ChecksumValue:
+      - null
+      - $.Checksum[0].Value
+      attr_ptype:
+      - null
+      - $.attr_ptype
+      attr_serial_identif:
+      - null
+      - $.attr_serial_identif
+      attr_platform_short_name:
+      - null
+      - $.attr_platform_short_name
+      attr_processing_date:
+      - null
+      - $.attr_processing_date
+      attrs:
+      - null
+      - $.Attributes
+      href:
+      - null
+      - http://127.0.0.1:5000/Products({id})/$value
+      geometry: POLYGON((180 -90, 180 90, -180 90, -180 -90, 180 -90))
+      downloadLink: http://127.0.0.1:5000/Products({id})/$value
+      title: $.Name
     free_text_search_operations:
       $filter:
         union: ' OR '
         wrapper: '"{}"'
         operations:
-            and:
-              - 'PublicationDate gt {startTimeFromAscendingNode#to_iso_utc_datetime}'
-              - 'PublicationDate lt {completionTimeFromAscendingNode#to_iso_utc_datetime}'
-              - "Attributes/OData.CSC.StringAttribute/any(att:att/Name eq 'productType' and att/OData.CSC.StringAttribute/Value eq '{attr_ptype}')"
-              - "Attributes/OData.CSC.StringAttribute/any(att:att/Name eq 'platformSerialIdentifier' and att/OData.CSC.StringAttribute/Value eq '{attr_serial_identif}')"
-              - "Attributes/OData.CSC.StringAttribute/any(att:att/Name eq 'platformShortName' and att/OData.CSC.StringAttribute/Value eq '{attr_platform_short_name}')"
-              - "Attributes/OData.CSC.StringAttribute/any(att:att/Name eq 'processingDate' and att/OData.CSC.StringAttribute/Value eq '{attr_processing_date#to_iso_utc_datetime}')"
-
+          and:
+          - PublicationDate gt {startTimeFromAscendingNode#to_iso_utc_datetime}
+          - PublicationDate lt {completionTimeFromAscendingNode#to_iso_utc_datetime}
+          - Attributes/OData.CSC.StringAttribute/any(att:att/Name eq 'productType'
+            and att/OData.CSC.StringAttribute/Value eq '{attr_ptype}')
+          - Attributes/OData.CSC.StringAttribute/any(att:att/Name eq 'platformSerialIdentifier'
+            and att/OData.CSC.StringAttribute/Value eq '{attr_serial_identif}')
+          - Attributes/OData.CSC.StringAttribute/any(att:att/Name eq 'platformShortName'
+            and att/OData.CSC.StringAttribute/Value eq '{attr_platform_short_name}')
+          - Attributes/OData.CSC.StringAttribute/any(att:att/Name eq 'processingDate'
+            and att/OData.CSC.StringAttribute/Value eq '{attr_processing_date#to_iso_utc_datetime}')
     pagination:
       max_items_per_page: 1000
       next_page_url_tpl: '{url}?{search}&$top={items_per_page}&$expand=Attributes'
-      total_items_nb_key_path: '$.total'
-
+      total_items_nb_key_path: $.total
   download:
     type: HTTPDownload
-    base_uri: 'http://127.0.0.1:5001/Products'
```

Comment:
@agrosu1978 @Padeanu why did we have 5001 here (that corresponds to the adgs2 station) instead of 5000 ? For pytests, or a mistake ?